src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
```suggestion
# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Reply:
As of #7066, the pre-commit hook will check the copyright year for you! You can enable the pre-commit hook with `./build-support/bin/setup.sh`. The pre-commit hook can occasionally take some time to run and can occasionally flake, and I'm working on adding testing in #7069 so that this gets better -- please let me know if you have any issues!